packages/app/src/elements/app.ts

```diff
@@ -31,10 +31,10 @@ import "./rich-content";
 import { alertDisabledFeature, displayProvisioning, getDefaultStatusLabel } from "../lib/provisioning";
 import { ItemsView } from "./items";
 import { wait, throttle } from "@padloc/core/src/util";
-import { AuditMixin } from "../mixins/audit";
+import { auditVaults } from "../lib/audit";
```

Comment:
I realised that the way we're doing things now, there is really no value in implementing this as a mixin as opposed to just exporting a function.